samples/kubernetes/deployment.yaml

```diff
@@ -1,3 +1,16 @@
+# Secret used to set up HTTP basic auth between the service broker and the service catalog.
+apiVersion: v1
+kind: Secret
+metadata:
+  name: atlas-service-broker-auth
+type: Opaque
+stringData:
+  username: username
+  password: password
+
+---
+
```

Comment:
Would remove this blank line—to me it implies that the dashes are a separator when they're not.

Reply:
Fixed by formatter